debian/postinst

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+# postinst script for nss-envhosts
+#
+# see: dh_installdeb(1)
+
+set -e
+
+case "$1" in
+    configure)
+    ;;
+
+    abort-upgrade|abort-remove|abort-deconfigure)
+    ;;
+
+    *)
+        echo "postinst called with unknown argument \`$1'" >&2
+        exit 1
+    ;;
+esac
+
+# dh_installdeb will replace this with shell code automatically
+# generated by other debhelper scripts.
+
+#DEBHELPER#
+
+ldconfig
```

Comment:
You don't need to do this manually. `dpkg-trigger` handles it for you. If it's not, check that `dh_makeshlibs` is running as expected at packaging time and finding the library:

https://manpages.debian.org/testing/debhelper/dh_makeshlibs.1.en.html

Reply:
Ah, so that's why it works. I was wondering why it worked without me running it manually when I was testing some changes. I got rid of it and validated again that it works.